src/command_modules/azure-cli-acr/README.rst

```diff
@@ -91,7 +92,10 @@ Update a container registry
         --admin-enabled       : Indicates whether the admin user is enabled.
 		                        Allowed values: false, true.
         --resource-group -g   : Name of resource group.
-        --storage-account-name: The name of an existing storage account.
+        --storage-account-name: Default: A new storage account will be created. 
```

Comment:
@SteveLasker Can you remove this line since `acr update` command doesn't default to create a new storage account. Otherwise, LGTM.